src/contexts/ProviderContext.js

```diff
@@ -0,0 +1,37 @@
+import React, { createContext, useState } from 'react';
+export const ProviderContext = createContext({
+  doctorList: () => {},
+  setDoctorList: () => {},
+  filteredDoctorList: () => {},
+  setFilteredDoctorList: () => {},
+  mentalHealthProsList:  () => {},
+  setMentalHealthProsList: () => {},
+  filteredMentalHealthProsList: () => {},
+  setFilteredMentalHealthProsList: () => {}
+});
+
+export const ProviderContextProvider = (props) => {
+
+  const [doctorList, setDoctorList] = useState([])
+  const [filteredDoctorList, setFilteredDoctorList] = useState([])
+
+  const [mentalHealthProsList, setMentalHealthProsList] = useState([])
+  const [filteredMentalHealthProsList, setFilteredMentalHealthProsList] = useState([])
+
+  return (
```

Comment:
Clever solution for getting the SearchBar working on all needed pages with whatever data it needs and more control over styling!